### ISSUE TITLE
Navbar: Option to Configure Profile menu Signout button text 

### ISSUE THREAD

**Issue by VinoSakthiv** [trimble-oss/modus-web-components#1603]
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-web-components/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-web-components/blob/main/CONTRIBUTING.md)

### Describe the issue

Profile Menu Signout button should be configurable to provide custom text

<img width="409" alt="image" src="https://github.com/trimble-oss/modus-web-components/assets/133335850/005d7dbe-afdc-4c4e-8b6f-fda26f14a1e1">

### What is the issue regarding ?

@trimble-oss/modus-web-components

### Priority

High

### What product/project are you using Modus Components for ?

Trimble Creator (Drawings)

### What is your team/division name ?

Industry Clouds

### Are you willing to contribute ?

Yes

### Are you using Modus Web Components in production ?

Yes

**Comment by VinoSakthiv** [comment]
@cjwinsor Yes during the process of internationalization in [Construction cloud UI](https://platform.trimble.com/app/)  we have identified this 'Sign out' text could not be localized. Currently we support Japanese language other than default English. and for the most part we have translated and some are yet to be translated
we have created this ticket to track the missing translations

**Comment by cjwinsor** [comment]
Add property to overwrite the Sign Out text, defaulting to standard.

**Comment by cjwinsor** [comment]
@VinoSakthiv Feel free to contribute this change, it has been approved.
